routers/install/install.go

```diff
@@ -385,7 +385,6 @@ func SubmitInstall(ctx *context.Context) {
 		ctx.RenderWithErr(ctx.Tr("install.invalid_db_setting", err), tplInstall, &form)
 		return
 	}
-	db.UnsetDefaultEngine()
```

Comment:
woder why unset? is it necessary? how about move it line 583 (before ``srv.Shutdown()`` )